app/views/devise/registrations/new.html.erb

```diff
@@ -13,6 +13,12 @@
   <%= f.text_field :name, class:"input_field input"  %>
 </div>
 
+  <div class="field">
+  <%= f.label :role %><br />
+  <h3>Choose: User or Admin</h3>
+  <%= f.text_field :role, class:"input_field input"  %>
```

Comment:
- [ ] The registration view for devise doesn't work, kindly follow the requirements for this project and fix this issue. The issue of `roles` will come in another milestone, for now, you can just style the views to match the wireframe. Your database doesn't have an attribute with the name `role`.

![role](https://github.com/Kaghenimbale/blog-app/assets/53124548/9fbf30cf-24d3-4306-aa51-12c57d779a55)

Reply:
@elisha2kyakpo1 can we meet?

Reply:
@Kaghenimbale, when you take out the code having `role` attribute, the app works.

Reply:
can you join this meeting so that you can explain https://microverse.zoom.us/j/91760234327?pwd=V25ja1JJQkhSTkJhQTI3NWl5QlJGdz09 because everything is working fine here

Reply:
@elisha2kyakpo1 I just remain with 1 review or @addod19 can you help?